spec/features/lettings_log_spec.rb

```diff
@@ -88,14 +88,115 @@
+            expect(page).not_to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+            visit("lettings-logs/#{log_id}/setup/check-answers")
+            expect(page).not_to have_content("Managing agent ")
+          end
+        end
+
+        context "and the owning organisation has 1 or more managing agents" do
+          let!(:managing_org1) { create(:organisation, name: "Managing org 1") }
+          let!(:org_rel1) { create(:organisation_relationship, parent_organisation: support_user.organisation, child_organisation: managing_org1) }
+
+          it "does show the managing organisation question" do
+            visit("/lettings-logs")
+            click_button("Create a new lettings log")
+            click_link("Set up this lettings log")
+            log_id = page.current_path.scan(/\d/).join
+            select(support_user.organisation.name, from: "lettings-log-owning-organisation-id-field")
+            click_button("Save and continue")
+            expect(page).to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+            select(managing_org1.name, from: "lettings-log-managing-organisation-id-field")
+            click_button("Save and continue")
+            visit("lettings-logs/#{log_id}/setup/check-answers")
+            expect(page).to have_content("Managing agent Managing org 1")
+          end
+
+          context "and the owning organisation has 2 or more managing agents" do
+            let!(:managing_org2) { create(:organisation, name: "Managing org 2") }
+            let!(:org_rel2) { create(:organisation_relationship, parent_organisation: support_user.organisation, child_organisation: managing_org2) }
+
+            context "and the organisation relationship for the selected managing agent is deleted" do
+              it "doesn't change the CYA page text to be 'You didn't answer this question'" do
+                visit("/lettings-logs")
+                click_button("Create a new lettings log")
+                click_link("Set up this lettings log")
+                log_id = page.current_path.scan(/\d/).join
+                select(support_user.organisation.name, from: "lettings-log-owning-organisation-id-field")
+                click_button("Save and continue")
+                expect(page).to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+                select(managing_org1.name, from: "lettings-log-managing-organisation-id-field")
+                click_button("Save and continue")
+                visit("lettings-logs/#{log_id}/setup/check-answers")
+                expect(page).to have_content("Managing agent Managing org 1")
+                org_rel1.destroy!
+                visit("lettings-logs/#{log_id}/setup/check-answers")
+                expect(page).to have_content("Managing agent Managing org 1")
+                expect(support_user.organisation.managing_agents).to eq(org_rel2.child_organisation)
```

Comment:
Similarly to above, this line is pretty artificial/contrived way of explicitly referencing the organisation relationship in the test. Without it, Rubocop complains that something being defined with `let!` is not being explicitly used in the test. The reason I have defined the organisation relationship at all is because its existence influences the logic around the answer options for the question, and in particular whether the question gets auto-assigned an answer.